myproject/homeFunctions.php

```diff
@@ -0,0 +1,70 @@
+
+
+<?php
+echo "<br>ФУНКЦИИ<br>";
+
+echo "<br>1.<br>";  // вычисление скорости движения
+?>
+
+<form method="get">
+    Введите пройденный путь <input type="text" name="distance"><br>
+    Введите время движения<input type="text" name="time"><br>
+    Пройденный путь в км<input type="checkbox" name="km" value="Yes"><br>
+    Пройденный путь в м<input type="checkbox" name="m" value="Yes"><br>
+    Время движения в час.<input type="checkbox" name="hour" value="Yes"><br>
+    Время движения в сек.<input type="checkbox" name="sec" value="Yes"><br>
+    <input type="submit" name="submit" value="Рассчитать">
+    <input type="hidden" name="submit" value="yes"> 
```

Comment:
You have two fields with the same name. It's mistake. Use different names for different fields of one form.